src/tarballs/build.ts

```diff
@@ -66,7 +66,21 @@ export async function build(c: BuildConfig, options: {
     const yarnRoot = findYarnWorkspaceRoot(c.root) || c.root
     if (fs.existsSync(path.join(yarnRoot, 'yarn.lock'))) {
       await fs.copy(path.join(yarnRoot, 'yarn.lock'), path.join(c.workspace(), 'yarn.lock'))
-      await exec('yarn --no-progress --production --non-interactive', {cwd: c.workspace()})
+
+      const yarnVersion = (await exec('yarn -v')).stdout.charAt(0)
```

Comment:
is `yarn` guaranteed to be installed? This would throw a `command not found` error if not